File: Python PowerPoint Examples/DocumentApiDemo/getPptDocument.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey, baseUrl)
api = spirecloudpowerpoint.api.document_api.DocumentApi(configuration)

name = "GetPptDocument.pptx"
password = ""
storage = ""
folder = "input"

result = api.get_ppt_document(name, password=password, storage=storage, folder=folder)

File: PHP PowerPoint Examples/DocumentApiDemo/getPptDocument.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\DocumentApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new DocumentApi($configuration);

$name = "GetPptDocument.pptx";
$password = null;
$folder = "input";
$storage = null;
$result = $apiInstance->getPptDocument($name, $password, $storage, $folder);
print_r($result);